### PR TITLE
Set batch-2.0 and related features to beta

### PR DESCRIPTION
**Pull request by cgianfra**
Stage all Jakarta Batch related features for inclusion in beta
Add batch-2.0 and batchManagement-2.0 to EE9 convenience feature

### CODE REVIEW

**Comment by cgianfra** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/io.openliberty.jakartaee-9.0.feature [hunk 1]
```diff
@@ -13,6 +13,8 @@ Subsystem-Name: Jakarta EE Platform 9.0
  com.ibm.websphere.appserver.transaction-2.0,\
  io.openliberty.appAuthorization-2.0,\
  io.openliberty.appClientSupport-2.0,\
+ io.openliberty.batch-2.0,\
+ io.openliberth.batchManagement-2.0,\
```
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/io.openliberty.jakartaee-9.0.feature:17]: batchManagement was not part of the convenience feature in 8.  Is there a reason for it to be part of it in 9?  liberty is spelled wrong for the feature though.  If you want it to be in the beta image for Jakarta 9 and it doesn't belong in the convenience feature, you should add it to `dev/build.image/profiles/jakartaee9/features.xml`.  Confirm with @tevans78

File: dev/com.ibm.websphere.appserver.features/visibility/public/batchManagement-2.0/io.openliberty.batchManagement-2.0.feature [hunk 2]
```diff
@@ -27,6 +27,6 @@ Subsystem-Name: Batch Management 2.0
 -files=bin/batchManager.bat, \
```
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/visibility/public/batchManagement-2.0/io.openliberty.batchManagement-2.0.feature:27]: If you don't have batchManager 1.0 also in your environment, batchManager scripts won't work I believe.  As such maybe batchManager shouldn't be marked beta, yet?  Is it needed for the CTS?

**Review by jhanders34 — CHANGES_REQUESTED** [review]

**Comment by cgianfra** [comment]
#build

**Review by jhanders34 — APPROVED** [review]

**Comment by jhanders34** [comment]
#libby

**Pull request merged**